projects/mc_interface/test/test_motor_controller.c

```diff
@@ -73,8 +75,14 @@ static void prv_fake_velocity_measurement(MotorControllerCanId id, float velocit
 
 static void prv_handle_speed(int16_t speed_cms[], size_t num_speeds, void *context) {
   int16_t *speed_arr = context;
-  speed_arr[0] = speed_cms[0];
-  speed_arr[1] = speed_cms[1];
+  memcpy(speed_arr, speed_cms, sizeof(int16_t) * num_speeds);
```

Comment:
Maybe add a static assert that an element of `speed_arr` and `speed_cms` are the same size?